src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,36 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.*;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("v1/ecommercee")
```

Comment:
End points should contain api keyword in it
```suggestion
@RequestMapping("api/v1/ecommercee")
```